### DISCUSSION TITLE
Combo Oot, MM randomizer

### DISCUSSION THREAD

**Discussion by Kethku** [HarbourMasters/2ship2harkinian#555] [Ideas]
https://ootmm.com/

I imagine something like this is a fairly obvious feature request, but I figured id weigh in that this is one of the things Im most hopeful for with 2s2h especially now that both games have native ports.

Amazing work yall. Super excited to see where it goes from here

**Comment by TraeF1337** [comment]
Please make the dream come true 🙏

**Comment by Sachaqc** [comment]
This hasn't been upvoted enough apparently. Please make it happen
